c2corg_ui/templates/image/preview.html

```diff
@@ -0,0 +1,70 @@
+<%namespace file="helpers/detailed_images_attributes.html"
```

Comment:
That's not that simple. In preview, you have many parts from view that you don't want to have in preview. For instance:
* the ``base.html`` template inheriting https://github.com/c2corg/v6_ui/blob/preview/c2corg_ui/templates/waypoint/view.html#L7 (not sure you can have a ``if`` condition to include it.
* ditto for the various ``block``

And having many ``if preview`` will make the template even more complicated and hard to read.

Please note that the ``preview.html`` templates are not that big thanks to the use of the various helper functions that you have created :)
https://github.com/c2corg/v6_ui/blob/preview/c2corg_ui/templates/waypoint/preview.html
Perhaps we could improve a bit by using an helper for the description/summary block as well if possible https://github.com/c2corg/v6_ui/blob/preview/c2corg_ui/templates/waypoint/preview.html#L47-L65